pkg/sdk/validation/manifest/json_remote_implementation.go

```diff
@@ -3,13 +3,29 @@ package manifest
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"strings"
 
-	hubpublicgraphql "capact.io/capact/pkg/hub/api/graphql/public"
+	"capact.io/capact/internal/ptr"
+	"capact.io/capact/internal/regexutil"
+	gqlpublicapi "capact.io/capact/pkg/hub/api/graphql/public"
+	"capact.io/capact/pkg/hub/client/public"
 	"capact.io/capact/pkg/sdk/apis/0.0.1/types"
+
+	"github.com/dustin/go-humanize/english"
 	"github.com/pkg/errors"
 )
 
+const (
+	ocfPathPrefix       = "cap."
```

Comment:
I wonder whether such const (and also `cap.type.` and `cap.core.type.`) shouldn't be moved to `pkg/sdk/apis/0.0.1/types/types.extend.go` to make sure they are only once defined.

I know now they are used just here, but please consider moving these definitions. If not, hopefully later someone will move them once they are used in multiple places 🙂